client/src/component/serviceCard/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { shape, string } from 'prop-types';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import { CardActions } from '@material-ui/core';
+import useStyles from './style';
+import { BOOK } from '../../utils/router.constant';
+
+const ServiceCard = ({ service }) => {
+  const { img, name, description } = service;
+  const classes = useStyles();
+  const fakeImage =
+    'https://www.matrix.com/~/media/images/hair-color-gallery/hair-color-looks/highlights/brown-highlights/brown-highlights-1.jpg';
+  return (
+    <Card className={classes.card}>
+      <CardActionArea className={classes.root}>
+        <CardMedia>
+          <img className={classes.image} src={img || fakeImage} alt="service" />
+        </CardMedia>
+        <CardContent>
+          <Typography gutterBottom variant="h5" component="h2">
```

Comment:
why variant and component are different?

Reply:
check [https://material-ui.com/api/typography/](url)
component is the element type 
but varient is for style you con make them different or same according to ur requirement